### DISCUSSION TITLE
IS rename cooldown

### DISCUSSION THREAD

**Discussion by Hyronymos1337** [BG-Software-LLC/SuperiorSkyblock2#1258] [Implemented Feature Requests] [answered]
Add a cooldown to is renaming.

![image](https://user-images.githubusercontent.com/73142735/178945504-fb14e29c-3350-4ccf-a253-8549a7f8a9fb.png)

**Comment by OmerBenGera** [comment, marked as answer]
You are more than welcome to add a cooldown in the config file to the rename command.
